### ISSUE TITLE
Create a new Orderer to the RAFT cluster and add it to a network.

### ISSUE THREAD

**Issue by himanshulalarya** [hyperledger-labs/blockchain-automation-framework#1031]
### Steps:

 1) go through the hyperledger-fabric documentation for orderer node (https://hyperledger-fabric.readthedocs.io/en/release-1.4/raft_configuration.html).
 2) Perform a manual addition of orderer on test-network of hyperledger-fabric.
 3) Prepare crypto material for the new orderer node.
 4) Add TLS certificate of the new orderer node to System Channel.
 5) Fetch the latest configuration block as the genesis block for the new orderer.
 6) Bring up new orderer pod
 7) Add new orderer end point to System Channel
 8) Add TLS certificate to Application Channel
 9) Add new orderer end point to Application Channel
 10) Testing on 2.2 fabric network also
 11) Readme and operational guide should be created

### Scenario:
- Given that the fault tolerance is 3n+1 and the original setup is having 3 orderer nodes.  Add 2 new orderer nodes to the orderer cluster.  This 2 new orderer nodes will be added to handle the channel also. 

 (Reference: https://medium.com/@kctheservant/add-an-orderer-to-a-running-raft-based-orderer-cluster-b6a432da138c)

### Acceptance Criteria
1. Test the network on Fabric v2.2.0 and v1.4.4

**Comment by lakshyakumar** [comment]
Two orderer cannot be added in the existing fabric raft network as the consenters simultaneous in fabric network, as it is forbidden by fabric mechanism.
<img width="736" alt="updation of one or more consenter" src="https://user-images.githubusercontent.com/56861637/95185539-744ce100-07e6-11eb-9d55-575077238176.PNG">
also the second orderer can be added only after the leader election.
<img width="729" alt="no raft leader" src="https://user-images.githubusercontent.com/56861637/95185680-a52d1600-07e6-11eb-80bb-e76881e3e7a2.PNG">

**Comment by suvajit-sarkar** [comment]
On hold till bug #1094 is fixed
